specs/trust-store-trust-policy.md

```diff
@@ -201,7 +203,8 @@ Users who consume signed artifacts from OCI registries, or signed arbitrary blob
             "registryScopes": [ "*" ],
             "signatureVerification": {
               "level" : "strict",
-              "verifyTimestamp": "afterCertExpiry"  // Only verify any timestamp countersignature when codesigning certificate chain has expired
+              "verifyTimestamp": "afterCertExpiry",  // Only verify any timestamp countersignature when codesigning certificate chain has expired
+              "skipTimestampRevocationCheck": true   // Skip the TSA certificate chain revocation check during timestamp verification
```

Comment:
If `skipTimestampRevocationCheck` is a bool option, does it support audit mode and log each timestamp signature verification check?

Reply:
In what situation do we want the timestamp revocation check but continue on failures? If user does not care about the result why not just skip it and save the roundtrip?